.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: build
+
+on: 
+  push:
+    branches:
+      - muggle
+  pull_request:
+    branches:
+      - muggle
+
+jobs:
+
+  django-app:
+    
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    strategy:
+      matrix:
+        python-version: [3.6]
+
+    runs-on: ubuntu-latest
+    name: Python ${{ matrix.python-version }}    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip==18.1 setuptools==49.6.0 wheel
+        pip install -r requirements.txt
+        pip install flake8
+    - name: Verify MySQL connection from host
+      run: |
+        sudo apt-get install -y mysql-client libmysqlclient-dev
+        mysql --host 127.0.0.1 --port ${{ job.services.mysql.ports[3306] }} -uroot -proot -e "SHOW GRANTS FOR 'root'@'localhost'"
+        mysql_tzinfo_to_sql /usr/share/zoneinfo | mysql --host 127.0.0.1 --port ${{ job.services.mysql.ports[3306] }} -uroot -proot mysql
+    - name: Setup MySQL Database and table
+      env:
+        DB_HOST: 127.0.0.1
+        DB_PORT: ${{ job.services.mysql.ports[3306] }} 
+      run: |
+        mysql --host $DB_HOST --port $DB_PORT -uroot -proot -e "SHOW DATABASES"
+        mysql --host $DB_HOST --port $DB_PORT -uroot -proot -e "SET GLOBAL sql_mode = 'ONLY_FULL_GROUP_BY,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION'";
+        mysql --host $DB_HOST --port $DB_PORT -uroot -proot -e "CREATE DATABASE sortinghat_db"
+        mysql --host $DB_HOST --port $DB_PORT -uroot -proot -e "SHOW DATABASES"
```

Comment:
Not sure about this. The testing command creates by itself the database.

Reply:
The migrations step needs the `sortinghat_db` database.
```
$ python manage.py migrate --settings=config.settings.testing
```
Else it would raise the error `django.db.utils.OperationalError: (1049, "Unknown database 'sortinghat_db'")`
But, I understand that the migration step is not needed for running the test suite. I'll remove it and update the PR.
Thanks for pointing this.